### PR TITLE
chore: add integ tests section to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by nija-at**
----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -130,6 +130,37 @@ Work your magic. Here are some guidelines:
...
+1. Acts as a regression detector. It does this by running `cdk synth` on the integration test and comparing it against
+   the `*.expected.json` file. This highlights how a change affects the synthesized stacks.
+2. Allows for a way to verify if the stacks are still valid CloudFormation templates, as part of an intrusive change.
+   This is done by running `yarn integ` which will run `cdk deploy` across all of the integration tests in that package.
+   Remember to set up AWS credentials before doing this.
+3. (Optionally) Acts as a way to validate that constructs set up the CloudFormation resources as expected. A successful
+   CloudFormation deployment does not mean that the resources are set up correctly.
+
+If you are working on a new feature that is using previously unused CloudFormation resource types, or involves 
+configuring resource types across services, you need to write integration tests that use these resource types of
```
Comment (jogold) [inline CONTRIBUTING.md]:
```suggestion
configuring resource types across services, you need to write integration tests that use these resource types or
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -130,6 +130,37 @@ Work your magic. Here are some guidelines:
...
+features.
+
+To the extent possible, include a section (like below) in the integration test file that specifies how the successfully
+deployed can be verified for correctness. Correctness here implies that the resources have been set up correctly. The
```
Comment (jogold) [inline CONTRIBUTING.md]:
```suggestion
deployed stack can be verified for correctness. Correctness here implies that the resources have been set up correctly.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -130,6 +130,37 @@ Work your magic. Here are some guidelines:
...
+steps here are usually AWS CLI commands but they need not be.
```
Comment (jogold) [inline CONTRIBUTING.md]:
```suggestion
The steps here are usually AWS CLI commands but they need not be.
```

Comment (nija-at) [reply CONTRIBUTING.md]: `The` is in the previous line.

Comment (nija-at) [reply CONTRIBUTING.md]: My bad. I see your previous comment was on the previous line 😎

**Pull request merged**